iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -496,5 +515,23 @@ private static double GenerateTemperatureWithinRange(int max = 50, int min = 0)
         {
             return Math.Round(s_random.NextDouble() * (max - min) + min, 1);
         }
+
+        private async Task UpdateReportedPropertiesWhenWritablePropertiesEmptyAsync(string componentName, CancellationToken cancellationToken)
+        {
+            const string propertyName = "targetTemperature";
+            
+            var reportedProperties = new ClientPropertyCollection();
+
+            // If the writable properties for the current component are empty, report the default value with ACK(ac=203, av=0).
+            var propertyValue = _deviceClient.PayloadConvention.PayloadSerializer.CreateWritablePropertyResponse(
+                defaultPropertyValue, 203, 0);

```

Comment:
`av` is 0, representing that the acknowledgment version is not referring to any desired properties as they are empty now.

Reply:
Can we describe what 203 and 0 represent and the reasoning behind the ACK value we have chosen?

Reply:
> Can we describe what 203 and 0 represent and the reasoning behind the ACK value we have chosen?

0 represents that `av` is not referring to any desired properties set via the hub. When a desired property is set up, its version number starts from 1 by default. 
For 203, I can check with @rido-min for the reasoning though, and add comments into our samples further so it can be clearer to our users.

Reply:
we are describing these cases in the PnP convention